spec/fixtures/education_benefits_claims/0994/prefill.spl

```diff
@@ -0,0 +1,81 @@
+*INIT*
+TEST
+
+PERSONAGE
+
+123456789
+V0994
+
+
+VetTec
+*START*
+VA Form 22-0994
+JAN 2019
+
+APPLICATION FOR VETERAN EMPLOYMENT THROUGH TECHNOLOGY EDUCATION COURSES
+                   (VET TEC) HIGH TECHNOLOGY PROGRAM
+        -------------------------------------------------------
+
+                        APPLICANT INFORMATION
+                        ---------------------
+
+Name: Test Personage
+
+Social Security Number: 123456789
+
+Mailing Address:
+123 TEST ST
+APT B
+TEST, TN, 12345
+USA
+
+Sex of Applicant: F
+
+Date of Birth: 1970-01-01
+
+Email Address: test@test.com
+
+Day Time Telephone Number: 5551234567
+
+Night Time Telephone Number: N/A
+
+Have you ever applied for VA education benefits?: YES
+
+Are you currently on active duty?: YES
+
+Do you anticipate you will go on active duty during the
+VET TEC program?: N/A
+
+DIRECT DEPOSIT INFORMATION:
+
+Routing or Transit Number: *****2115
+
+Account Type: checking
+
+Account Number: *********1234
+
+                          COURSE OF STUDY
+                          ---------------
+
+VET TEC programs:
+
+                       BACKGROUND INFORMATION
+                       ----------------------
+
+Are you working in a high-tech field now?: N/A
+
+Have you worked in a high-tech industry in the past?: N/A
```

Comment:
Does this line need to be `industry`, or should it also change to `field`?

Reply:
..and it's merged before I hit F5, whoops.

Reply:
Yes, that's correct based on the UX-approved content doc.